app/controllers/returns/credits/ConvertedCreditsController.scala

```diff
@@ -61,11 +61,10 @@ class ConvertedCreditsController @Inject()
             formValue => {
               request.userAnswers
                 .setOrFail(ConvertedCreditsPage, formValue)
+                .setOrFail(WhatDoYouWantToDoPage, true)
```

Comment:
will this not/this should have been set by the WhatDoYouWantToDoController.onSubmit